lace/emulator/nn_emulator.py

```diff
@@ -457,7 +600,15 @@ def _obtain_sim_params(self):
             for i in range(len(self.training_data))
         ]
         training_label = np.array(training_label)
-        self.yscalings = np.median(training_label)
+                
+        if not self.emulator_label in ['Cabayol23', 'Cabayol23_extended']:
```

Comment:
Are computing the power spectrum summary statistics here? This should be done in the archives, in fact, we are already doing it.

Reply:
This is for the normalization. It has always been inside the emulator code. This has also been useful to test different scalings and by doing it within the emulator, we make sure that the conversion back is correct. Do you have anything agains keeping this here?

Reply:
Ah, ok! I thought it was the summary statistics, but I can see now that it is the polynomial fit. Forget about it :)